.ci/jenkins/project/Jenkinsfile.nightly

```diff
@@ -49,6 +53,21 @@ pipeline {
             }
         }
 
+        // Applies only to the 9.x branch build. The 9.?.x release branches must remain intact.
+        stage('Migrate to 9') {
+            when {
+                expression { return is9xBranch() }
+            }
+            steps {
+                script {
+                    checkoutRepo(optaPlannerRepository, optaPlannerMainBranch)
```

Comment:
leftover ?

Reply:
yeah but in `checkoutNewBranch` you will override this if I get it correctly

Reply:
I mean `checkoutRepo` and `checkoutNewBranch` are both checking out the `optaPlannerMainBranch` into the `optaPlannerRepository` directory

Reply:
I see now, thanks!